set.mm

```diff
@@ -528868,6 +528934,405 @@ Set induction (or epsilon induction)
       VNAVMVACVBVGUJABVGUEUFUGUHUIVEVKUKULABVGIVPRUMUNDVFUOUPUQABIURUS $.
   $}
 
+$(
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+  Transitive closure of a class
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+$)
+
+  $( Declare new constant $)
+  $c t++ $.
+
+  $( Declare the syntax for the transitive closure of a class.  See
+     ~ df-ttrcl . $)
```

Comment:
"See ~ df-ttrcl ." can be removed (I did not see such a reference to the definition anywhere else...).

Reply:
Done